_rules/SC1-1-1-image-accessible-name-is-placeholder-text.md

```diff
@@ -0,0 +1,78 @@
+---
+name: Image accessible name is placeholder text
+description: |
+  This rule checks that image elements that use generic placeholder text as their accessible name do so without loss of infomation to the user.
+
+success_criterion:
+- 1.1.1 # Non-text Content
+
+test_aspects:
+- DOM Tree
+- CSS Styling
+
+authors:
+- Bryn Anderson
+---
+
+### Applicability
+
+The rule applies to HTML `input type="image"` or `area` elements, or any HTML element with the semantic role of `img`, where the element is [exposed to assistive technologies](#exposed-to-assistive-technologies) and has an [accessible name](#accessible-name) that exclusively uses [generic placeholder text](#generic-placeholder-text).
+
+### Expectation
+
+Each test target has an accessible name that can be used in place of the [non-text content](https://www.w3.org/TR/WCAG21/#dfn-non-text-content) of the element without loss of information to the user.
```

Comment:
I don't know what loss of information means. I suggest using the wording of the success criterion: "The accessible name of each test target provides serves the equivalent purpose of the element" or such.

Reply:
The rule follows the same logic as #263 (which is written upon the same failure F30)

I think here (as in #263), the idea is to write a rule that can be converted in a semi-automated test : a semi-auto test could, based on conditions, flag what seems to be generic placeholder text, and wait for a human validation for the relevance of the generic text.